Blog Post: IPFS Camp Course Videos Released

terichadbourne opened the pull request:
Clearly not ready for a review, but sharing my code because it's my first time attempting to use this repo (I'm not familiar with Hugo) and I'm unable to make my blog post appear in localhost. I feel like I've followed the instructions in the readme, so perhaps there's a missing step there or a caveat I'm not grasping? 

I have: 
- created a new markdown file with the filename in the requested format
- updated the YAML top matter as indicated
- run `make dev` (lots of warnings but no errors)
- visited the blog homepage via `localhost:1313` and don't see the new post included in the index
- visited the url I believe is appropriate for the specific post and get a 404 message
- tried running `make` in case that would help 
- ran `make dev` again
- same results

![image](https://user-images.githubusercontent.com/19171465/64876022-42397180-d61c-11e9-8547-1718ab3390e2.png)
![image](https://user-images.githubusercontent.com/19171465/64876373-f9ce8380-d61c-11e9-97af-66f6ee0209ac.png)

@renrutnnej or anyone who's written for our blog before, could you help me figure out what I'm doing wrong to get this preview up and running locally?

terichadbourne commented:
I also now have a bunch of edits that seem to have been created by the `make` command which I assume I should discard? (I don't understand from the readme when someone should actually use `make` and not `make dev`.) 

@olizilla, @renrutnnej said I should ping you for technical assistance and mentioned she had a preview issue earlier in the week because of "some new token thing." 🤷‍♀️ Any ideas?

olizilla commented:
`make dev` for local dev preview which should "live reload" changes as you make them, vs `make` which is for building out the final static site ready to publish to ipfs. Let's take a look at your set up!

terichadbourne commented:
@olizilla and I just did some troubleshooting and discovered that it's not possible to preview blog posts with a date that's in the future, so authors can apparently only preview in advance by adjusting to a date  they know won't be the actual publication date. Seems like a nice safety feature for the live site but not very helpful for authors. (I'll update the readme.)

terichadbourne commented:
Not quite finished, but I'm sharing my rough draft here for feedback on the structure and tone.

terichadbourne commented:
I've finished the content here and would appreciate a detailed review, especially from @olizilla & @mikeal (in @alanshaw's absence) since their sessions are most called out here and I want to ensure I'm not mis-stating anything.

@momack2 Please let me know if this is in line with the vibe and content you had in mind. 

Before this is merged we need to: 
- [ ] Make videos public (not absolutely essential, but highly advisable) 
- [ ] Confirm what release date we want for the post dependent on video release plan (@doctorrobinson)
- [ ] Update the post date in the YAML header accordingly 
- [ ] Update the link for the Understanding How IPFS Deals with Files video, which @doctorrobinson is updating the title for

mikeal commented:
LGTM. I wasn’t able to see the other courses (because I was teaching a course) so I can’t comment on the second half very well.

jessicaschilling commented:
@terichadbourne, blog post is awesome! Question: maybe use "we" instead of "I" for consistency with how other blog posts are worded? I understand that posts have bylines attached to individuals, but most posts (particularly weekly newsletter posts) don't feel like they come from an individual person, but more from an overall IPFS team.

terichadbourne commented:
Thanks for the feedback @jessicaschilling. I'd be happy to change "I" to "we" if no one objects. I think there's only one spot I actually did it, which is where I'm recommending those two particular courses as a good starting point. I loved them because they felt the most beginner-friendly, but they're also (IMHO) the most easily consumable online because you don't need to do stuff other than watch the videos to take them in (versus the app-building workshops).

momack2 commented on content/post/057-ipfs-camp-course-videos.md:
```diff
@@ -0,0 +1,80 @@
+
+**Check out the [complete course listings](https://github.com/ipfs/camp#-core--elective-courses) in the IPFS Camp repo for links to videos, slide decks, and other workshop materials for each session.**
+
+Whether you're new to IPFS or looking for a deeper understanding of core concepts, I highly recommend these two core courses as your starting point. Both are easily consumable in video form -- no coding required -- and offer plenty of "aha" moments:
```
Agree w/ Jessica re shared voice
```suggestion
Whether you're new to IPFS or looking for a deeper understanding of core concepts, we highly recommend these two core courses as your starting point. Both are easily consumable in video form -- no coding required -- and offer plenty of "aha" moments:
```

momack2 commented on content/post/057-ipfs-camp-course-videos.md:
```diff
@@ -0,0 +1,80 @@
+
+- [Identity on the DWeb](https://youtu.be/EsLqjFEN-MM)
+- [Managing pinsets with IPFS Cluster](https://youtu.be/wmxaVrAFfeE)
+- [Cat Roulette!](https://youtu.be/kVzHE4MRPOM)
```
```suggestion
- [Cat Roulette! Building a p2p laser catbot](https://youtu.be/kVzHE4MRPOM)
```
If you just get the title, ideally make it sufficiently descriptive ;)

momack2 commented on content/post/057-ipfs-camp-course-videos.md:
```diff
@@ -0,0 +1,80 @@
+
+**Elective Courses**
+
+- [Identity on the DWeb](https://youtu.be/EsLqjFEN-MM)
```
```suggestion
- [Identity on the DWeb using IDM and Nomios](https://youtu.be/EsLqjFEN-MM)
```

momack2 commented on content/post/057-ipfs-camp-course-videos.md:
```diff
@@ -0,0 +1,80 @@
+- [Developing apps with IPFS API](https://youtu.be/6EHBOnFQJN4)
+- [The lifecycle of data in DWeb](https://youtu.be/fLUq0RkiTBA)
+
+**Elective Courses**
```
should we say who was involved in each of these courses? in most cases it's clear (cluster, textile, qri, etc), but the infra one in particular should give credit to @obo20 for the great presentation!

terichadbourne replied:
Thanks for the suggestion @momack2. Are you thinking that we should list all of the credited teammates for each preso (as found in the repo and the video title slides)? I'm under the impression that some were very much a team effort as indicated there and some ended up much more on one person to create, in which cases I wouldn't know who that was for most of these. 🤷‍♀️

terichadbourne replied:
Just pushed an edit that adds all of the authors listed in the camp repo for each course.

terichadbourne commented:
I just: 
- added course authors' handles
- updated publication date to today
- made title caps consistent
- updated video links for the dealing with files course

To do before merge: 
- [ ] Proof from @olizilla 
- [ ] Proof from @doctorrobinson
- [ ] Videos pushed live on YouTube by @doctorrobinson
- [ ] Merge https://github.com/ipfs/camp/pull/184 (adding video links to camp repo) once it's reviewed
- [ ] Update publication date in YAML `date` and `URL` fields if publication date is not 9/17

doctorrobinson commented:
Aww I can't check boxes for you! @terichadbourne all the videos are live, did a review of your content, everything looks good to me!

doctorrobinson commented on content/post/057-ipfs-camp-course-videos.md:
```diff
@@ -0,0 +1,80 @@
+
+Watch the recording below or check out the [slides](https://github.com/ipfs/camp/blob/master/CORE_AND_ELECTIVE_COURSES/CORE_COURSE_A/IPFS_Camp_Core_Course_A_Slides.pdf)!
+
+[![Video: Understanding How IPFS Deals with Files](/img/057-ipfs-camp-course-videos/core-a-thumbnail.png)](https://youtu.be/dJ7TkB4UpZ8)
```
This image may need an update to the new one: https://drive.google.com/open?id=1sYpsu-K6DSSB81EICTlKXz0lQI_7pHBf

terichadbourne replied:
done, thanks for the reminder!

doctorrobinson commented on content/post/057-ipfs-camp-course-videos.md:
```diff
@@ -0,0 +1,80 @@
+The ProtoSchool curriculum is growing, and we're excited to explore all of the fabulous IPFS Camp content to find gems to add to our tutorials. Watch [this issue](https://github.com/ProtoSchool/protoschool.github.io/issues/261) to track those content developments, or learn more about [contributing to ProtoSchool](https://proto.school/#/contribute) as a tutorial author or chapter leader.
+
+## But wait, that's not all!
+There's even more great content from IPFS Camp content coming your way, including recordings of keynotes and interviews. If you're interested in receiving an update when the next batch of videos are available, try out one of these strategies:
```
Big fan of this plug! Do we do this everywhere? Because we should...

terichadbourne replied:
If you mean the references to how to subscribe to updates, then yes, I stole this from other recent camp-related posts. :)

terichadbourne commented:
This is good to go if it's going today, but I don't know what time of day we usually release posts or who's in charge around here. 😂  

If we prefer to release tomorrow it for different time zone coverage then it will need the date updated.

momack2 commented:
Ship it! Looks great to me. =]

terichadbourne commented:
Shipped! (I updated the date and reduced image size to ensure people would continue reading past the ginormous thumbnails.)

momack2 commented:
YAY!!! Tweet and we'll retweet from IPFSbot?

On Wed, Sep 18, 2019 at 8:45 AM Teri Chadbourne <notifications@github.com>
wrote:

> Shipped! (I updated the date and reduced image size to ensure people would
> continue reading past the ginormous thumbnails.)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ipfs/blog/pull/300?email_source=notifications&email_token=AAEXAF44VADYREWE7IJHCZ3QKJEH5A5CNFSM4IWRRZBKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD7AQXGQ#issuecomment-532745114>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEXAF6TLBDCBYDTBBGJLSTQKJEH5ANCNFSM4IWRRZBA>
> .
>

terichadbourne commented:
https://twitter.com/EventTeri/status/1174392753326895104


On Wed, Sep 18, 2019 at 11:58 AM MollyM <notifications@github.com> wrote:

> YAY!!! Tweet and we'll retweet from IPFSbot?
>
> On Wed, Sep 18, 2019 at 8:45 AM Teri Chadbourne <notifications@github.com>
> wrote:
>
> > Shipped! (I updated the date and reduced image size to ensure people
> would
> > continue reading past the ginormous thumbnails.)
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/ipfs/blog/pull/300?email_source=notifications&email_token=AAEXAF44VADYREWE7IJHCZ3QKJEH5A5CNFSM4IWRRZBKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD7AQXGQ#issuecomment-532745114
> >,
> > or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AAEXAF6TLBDCBYDTBBGJLSTQKJEH5ANCNFSM4IWRRZBA
> >
> > .
> >
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/ipfs/blog/pull/300?email_source=notifications&email_token=AESIRCN6VERBTLXTWVHTVOTQKJFZ3A5CNFSM4IWRRZBKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD7ASDQY#issuecomment-532750787>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AESIRCI4ATQ4MKYRT4A2FU3QKJFZ3ANCNFSM4IWRRZBA>
> .
>


-- 
*Teri Chadbourne, CMP*
Community Manager & Lead Maintainer, ProtoSchool <https://proto.school>
Protocol Labs <https://protocol.ai/>

Twitter <https://twitter.com/eventteri> | GitHub
<https://github.com/terichadbourne>
